Assets/Scripts/CameraController.cs

```diff
@@ -7,10 +7,16 @@
 public delegate void CameraMovementFunction();
 
 public class CameraController : MonoBehaviour {
+    enum ViewMode
+    {
+        Shooter,
+        Third_Person
+    }
     [Header("Linked Components")]
     public InputManager input_manager;
     public Camera controlled_camera;
     public GameObject home;
+    public GameObject home_model;

```

Comment:
The thing you wanted before was `GetComponentsInChildren`. So you could probably get rid of this and use `GetComponentsInChildren<Renderer>` instead. That way we fade everything that's a child of the player.